Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2003-09-23
Initial Package Version: alpha9.8
Origin: Mandrake CVS
Description: Missing Includes
 
$LastChangedBy$
$Date$

diff -Naur cdparanoia-III-alpha9.8.orig/header.c cdparanoia-III-alpha9.8/header.c
--- cdparanoia-III-alpha9.8.orig/header.c	1999-08-15 22:34:58.000000000 +0000
+++ cdparanoia-III-alpha9.8/header.c	2003-09-24 04:42:32.000000000 +0000
@@ -9,6 +9,7 @@
 
 #include <unistd.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 
 static void PutNum(long num,int f,int endianness,int bytes){
diff -Naur cdparanoia-III-alpha9.8.orig/paranoia/gap.c cdparanoia-III-alpha9.8/paranoia/gap.c
--- cdparanoia-III-alpha9.8.orig/paranoia/gap.c	2000-04-19 22:41:04.000000000 +0000
+++ cdparanoia-III-alpha9.8/paranoia/gap.c	2003-09-24 04:42:32.000000000 +0000
@@ -6,6 +6,7 @@
  *
  ***/
 
+#include <string.h>
 #include "p_block.h"
 #include "cdda_paranoia.h"
 #include "gap.h"
